server/users.go

```diff
@@ -11,10 +11,11 @@ import (
 )
 
 type userRequest struct {
-	ID       uint64 `json:"id,string"`
-	Name     string `json:"name"`
-	Provider string `json:"provider"`
-	Scheme   string `json:"scheme"`
+	ID       uint64   `json:"id,string"`
+	Name     string   `json:"name"`
+	Provider string   `json:"provider"`
+	Scheme   string   `json:"scheme"`
+	Roles    []string `json:"roles"`
```

Comment:
Why not []chronograf.Role?

Reply:
Good catch - this is vestigial from when we were adding more Permissions info. Thanks.

Reply:
No that isn't preferable because `["Viewer"]` locks you into an array of strings.  That is not extensible (see queryConfig as an example why this is bad).

You pointed out a few times that you aren't entirely sure what the final concept for multitenancy is going to be.  The way to handle that issue is to imagine after you write, say 20 lines of code, what is too rigid.  In the case of JSON, arrays of strings are nearly always too rigid.

Many times one wants a way to give more information or specificity to something as a program grows. In JSON you do that by making arrays of objects.